### PR TITLE
Increase Annotator resolution to 2048

### PR DESCRIPTION
**Pull request by Lissanro**
Solves issue #121

### CODE REVIEW

**Comment by ljleb** [comment]
I see some minimum values set to `minimum=128` instead of `minimum=64`, is that intentional? I don't know about the specifics of the preprocessors, just making sure all values are set correctly.

**Comment by Lissanro** [comment]
Yes, I changed the minimum values to match to the same minimum value in Width and Height sliders in the main section of A1111 UI. I can adjust those in 64-2048 range, and since preprocessor image will be either downscaled or upscaled if it does not match and scaled it can cause issues, this is why it is important to have an option to set Annotator resolution to the same value which is used for output Width/Height. If the Annotator size slider is limited to 1024, there is no way to set the correct value if my output size is for example 1536 or 2048.

**Comment by Lissanro** [comment]
I will update my pull request to resolve the conflict in a moment.

File: scripts/controlnet.py [hunk 1]
```diff
@@ -189,56 +189,56 @@ def refresh_all_models(*inputs):
...
                         return [
-                            gr.update(label="Normal Resolution", value=64, minimum=64, maximum=1024, interactive=False),
+                            gr.update(label="Normal Resolution", value=64, minimum=64, maximum=2048, interactive=False),
                             gr.update(label="Threshold A", value=64, minimum=64, maximum=1024, interactive=False),
                             gr.update(label="Threshold B", value=64, minimum=64, maximum=1024, interactive=False),
                         ]
                     else:
                         return [
-                            gr.update(label="Annotator resolution", value=512, minimum=64, maximum=1024, step=1, interactive=True),
+                            gr.update(label="Annotator resolution", value=64, minimum=64, maximum=2048, interactive=False),
```
Comment (Mikubill) [inline scripts/controlnet.py:234]: Use `value=512` and `interactive=True` for any other preprocessors might be better

**Pull request merged**